### PR TITLE
Restrict access to standard global IP addresses.

### PR DESCRIPTION
**Pull request by bemasc**

### CODE REVIEW

**Comment by bemasc** [comment]
This is a fix for https://github.com/uProxy/uproxy/issues/253

File: src/rtc-to-net/rtc-to-net.ts [hunk 1]
```diff
@@ -299,6 +317,20 @@ module RtcToNet {
...
 
+    private isAllowedAddress_ = (addressString:string) : boolean => {
+      // ipaddr.process automatically converts IPv4-mapped IPv6 addresses into
+      // IPv4 Address objects.  This ensure that an attacker cannot reach a
+      // restricted IPv4 endpoint that is identified by its IPv6-mapped address.
+      try {
+        var address = ipaddr.process(addressString);
+        return address.range() == 'unicast';
+      } catch (e) {
+        log.error(this.longId() + ': error in isAllowedAddress_(' +
```
Comment (iislucas) [inline src/rtc-to-net/rtc-to-net.ts]: Not sure we want to error here. I'd leave just the error at the `handleWebRtcControlMessage_`

**Comment by iislucas** [comment]
Looks great, just one comment that we probably don't need to double-report error. Otherwise :+1: Fix and merge on!

File: src/rtc-to-net/rtc-to-net.ts (hunk 1 above)
Comment (bemasc) [inline src/rtc-to-net/rtc-to-net.ts]: Done.

**Pull request merged**